benchmark_runner/common/clouds/IBM/assisted_installer_latest_version.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+# import urllib library
+from urllib.request import urlopen
+
+
+class OCPVersions:
+    """
+    This class get the latest assisted installer version from https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph
+    """
+    # MAJOR.MINOR.PATCH.BUILD
+    MAJOR = 0
+    MINOR = 1
+    PATCH = 2
+    BUILD = 3
```

Comment:
This is a bit confusing when referenced in code; `self.MAJOR` looks like the major version, not the component index.  Maybe something like `I_MAJOR` or `C_MAJOR` and so forth.